gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/ReevaluateDagProc.java

```diff
@@ -73,6 +75,7 @@ protected void act(DagManagementStateStore dagManagementStateStore, Pair<Optiona
       // dag actions for each of those parallel job and in this scenario there is no job status available.
       // If the job status is not present, this job was never launched, submit it now.
       DagProcUtils.submitJobToExecutor(dagManagementStateStore, dagNode, getDagId());
+      dagProcEngineMetrics.markDagActionsAct(DagActionStore.DagActionType.REEVALUATE, true);
```

Comment:
how about all dag procs implement the above mentioned `getDagActionType` method? looks like the same line can be used in many places

Reply:
replaced all of these with `getDagActionType()`